MANIFEST.in

```diff
@@ -2,7 +2,5 @@ include HISTORY.rst
 include LICENSE.txt
 include README.rst
 include pyproject.toml
-global-include *.pyx
-global-include *.pxd
-global-include *.h
 include requirements.txt
+recursive-include src *.h *.cpp *.pyx *.pxd
```

Comment:
OK, so this is just including the files with stated extensions from src. Seems like a good idea.

Reply:
Yep, that's the idea!  I never thought that `global-include` would reach inside a virtual environment, but there you have it.